tracer/src/Datadog.Trace/Agent/IApiResponse.cs

```diff
@@ -64,5 +72,48 @@ public static bool ShouldRetry(this IApiResponse response)
 
             return shouldRetry;
         }
+
+        internal static bool HasMimeType(string rawContentType, string mimeType)
+        {
+            if (string.IsNullOrEmpty(mimeType))
+            {
+                ThrowHelper.ThrowArgumentNullException(nameof(mimeType));
```

Comment:
Nit picking, but should we throw `ArgumentNullException` when `mimeType` is empty (not null)? We should probably throw a different exception.